hack/generate.sh

```diff
@@ -158,14 +158,20 @@ function generate_manifests() {
         kustomize build ${controller_crd_path} > ${controller_crd_path}/resources.yaml
         controller-gen object:headerFile=${hack_boilerplate} paths="./..."
         goimports -w  ${controller_path}
+        cd ..
     fi
 
     cp ${controller_crd_path}/resources.yaml ${BUILD_FOLDER}/resources.yaml
 }
 
 function generate_bundle() {
     ENABLE_KUBE_API=true generate_manifests
-    operator-sdk generate kustomize manifests --apis-dir api -q
+    # temp copy for operator-sdk that doesn't know how to handle sub-modules
```

Comment:
Maybe replace with a TODO so it will be explicit that we want it as temporary solution

Reply:
Unless someone have a better solution it will not be temporary, the load function that is called by operator-sdk is not configurable https://github.com/kubernetes-sigs/kubebuilder/blob/cf1575e08dff6c2b63d442b8fa6ec397fba64363/pkg/config/store/yaml/store.go#L65-L67

Reply:
maybe we can submit an issue for ``operator-sdk`` to handle that (including handling their dependencies)

Reply:
good idea https://github.com/operator-framework/operator-sdk/issues/5583